x-pack/plugin/autoscaling/src/main/java/org/elasticsearch/xpack/autoscaling/storage/ReactiveStorageDeciderService.java

```diff
@@ -722,6 +793,8 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.field("reason", reason);
             builder.field("unassigned", unassigned);
             builder.field("assigned", assigned);
+            builder.field("unassigned_shard_ids", unassignedShardIds);
+            builder.field("assigned_shard_ids", assignedShardIds);
```

Comment:
On second thought, perhaps we should have a count field and a truncated list of shard-ids instead, to use a json array for the list rather than a comma separated string.